JustSaying/NoOpMessageResponseLogger.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading.Tasks;
+using JustSaying.AwsTools.MessageHandling;
+using JustSaying.Models;
+
+namespace JustSaying
+{
+    internal class NoOpMessageResponseLogger : IMessageResponseLogger
```

Comment:
This isn't _strictly_ no-op, as it has properties set to `null`, rather than delegates that just do nothing.

Reply:
Forgot to rename that after switching them to nulls!

Reply:
There we go 👍

Reply:
Renamed to `DefaultMessageResponseLogger`.